docs/overview/office-add-ins-community-call.md

```diff
@@ -11,7 +11,7 @@ The Microsoft Office Add-ins community call is a monthly call where you can lear
 
 ## Agenda for October 12th call
 
-The agenda for the October call will be posted about a week before the call.
+- Upgrade your Visual Studio template for Office Add-ins.
 
 ## Previous calls
```

Comment:
Consider adding a link to the [September blog post](https://pnp.github.io/blog/office-add-ins-community-call/2022-09-14/).

Reply:
Added